src/ajax/check_hgvs_dialogue.php

```diff
@@ -56,48 +64,23 @@
     // Resetting the transcript fields.        
     var oTranscriptFields = $(\'#variantForm input[name$="VariantOnTranscript/DNA"]\');
     oTranscriptFields.val("").removeClass();
-    oTranscriptFields.siblings("img").attr({src: "gfx/trans.png"}).show();
+    oTranscriptFields.siblings("img").attr({src: "gfx/trans.png"});
     $(\'#variantForm input[name$="VariantOnTranscript/RNA"]\').val("").removeClass();
     $(\'#variantForm input[name$="VariantOnTranscript/Protein"]\').val("").removeClass();
     
     // Resetting the genomic fields.
     var oGenomicVariants = $(\'#variantForm input[name^="VariantOnGenome/DNA"]\');
     oGenomicVariants.val("").removeClass();
-    oGenomicVariants.siblings("img").attr({src: "gfx/trans.png"}).show();
+    oGenomicVariants.siblings("img").attr({src: "gfx/trans.png"});
 }
 ');
 
 
-if (!$sVariant) {
-    // If the variant is empty, we can simply close the script.
-    exit;
-}
-
-
-
-// Preparing variables to allow easy and readable updates of the dialogue.
-
-// Preparing the steps.
-$sStepInitialChecks = 'statusChecks';
-$sStepMapping       = 'statusMapping';
-
-// Preparing the images.
-$sImageNeutral = 'gfx/trans.png';
-$sImagePassed  = 'gfx/check.png';
-$sImageFailed  = 'gfx/cross.png';
-$sImageLoading = 'gfx/lovd_loading.gif';
-
-// Preparing the buttons.
-$sButtonYes            = 'oButtonYes';
-$sButtonNo             = 'oButtonNo';
-$sButtonOKValid        = 'oButtonOKValid';
-$sButtonOKInvalid      = 'oButtonOKInvalid';
-$sButtonOKCouldBeValid = 'oButtonOKCouldBeValid';
 
 // Preparing the JS for the buttons.
 print('
 // Preparing the buttons.
-var ' . $sButtonYes . ' = {"Yes":function () {
+var ' . 'oButtonYes' . ' = {"Yes":function () {
```

Comment:
Could you please remove the `' . '` string concatenations? :grimacing: 
(here and a few times below)
Thanks :sweat_smile: